### PR TITLE
update save action guide

### PR DESCRIPTION
**Pull request by chengren311**
See https://github.com/azkaban/azkaban/issues/1920

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,20 +53,21 @@ Install and enable the intellij's
...
-* Reformat code based on the code style
-* Rearrange code (reorder methods, fields, etc.)
-* Add final to local variable
-* Add final to field
-* Remove explicit generic type for diamond
-* Qualify field access with this
-* Remove final from private method
-* Remove unnecessary semicolon
-* Add missing @Override annotations
+Please turn on the following options only:
```
Comment (HappyRay) [inline CONTRIBUTING.md]: Specify the minimum version of the plugin that has these options?

Comment (chengren311) [reply CONTRIBUTING.md]: updated, thanks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -53,20 +53,21 @@ Install and enable the intellij's
...
-* Reformat code based on the code style
-* Rearrange code (reorder methods, fields, etc.)
-* Add final to local variable
-* Add final to field
-* Remove explicit generic type for diamond
-* Qualify field access with this
-* Remove final from private method
-* Remove unnecessary semicolon
-* Add missing @Override annotations
+Please turn on the following options only(all of them are available in release 1.1.0 and above, see https://github.com/dubreuia/intellij-plugin-save-actions/releases):
```
Comment (kunkun-tang) [inline CONTRIBUTING.md]: Just a quick typo fix: parentheses need a space ahead.

**Comment by HappyRay** [comment]
@juhoautio what do you think of the proposed conventions?

**Comment by juhoautio** [comment]
👍 This looks really good.

Any plans to apply these rules on all classes in the repo?

For example, if I apply save actions with this configuration on ExecutorManager.java, there are many references to the `logger` in the inner classes that then become `ExecutorManager.logger`.

**Comment by chengren311** [comment]
@juhoautio yes, once merged, reformatting the whole repo will be needed to be done.

**Comment by HappyRay** [comment]
> For example, if I apply save actions with this configuration on ExecutorManager.java, there are many references to the logger in the inner classes that then become ExecutorManager.logger.

I believe it won't happen with the proposed rule above:
Add class qualifier to static member access **outside declaring class only**

**Comment by chengren311** [comment]
@HappyRay actually class qualifier will be added if it's inside the inner class of declaring class.

**Comment by HappyRay** [comment]
I see. Do these inner classes need to share the parent class's logger?

**Comment by chengren311** [comment]
@HappyRay some inner class are used just for encapsulation purpose, but for the inner classes in ExecutorManager, i think it makes more sense to separate them out as regular classes.

**Comment by HappyRay** [comment]
I need to review the code to remember what they do. But a good thought. Care to refractor it?

**Comment by chengren311** [comment]
sure, @juhoautio FYI.

**Comment by juhoautio** [comment]
@chengren311 @HappyRay see my other pr, I already started doing that.

**Pull request merged**